pkg/internal/apis/config/validate.go

```diff
@@ -130,6 +140,68 @@ func (n *Node) Validate() error {
 	return nil
 }
 
+func formatPortMapping(addr net.IP, port int, protocol PortMappingProtocol) string {
+	return fmt.Sprintf("%s/%s",
+		net.JoinHostPort(addr.String(), strconv.Itoa(port)),
+		protocol,
+	)
+}
+
+func validatePortMapping(portMappingsSet sets.String, addrSet sets.String, mapping PortMapping) error {
```

Comment:
probably should be pointers to `sets.String`?

Reply:
`sets.String` is a map, and will be passed as a reference only 😇

Reply:
I guess we can depend on that since we have our own forked copy. If we were using a third-party set type I wouldn't depend on that detail.